scripts/CustomMaterials.zs

```diff
@@ -9,46 +9,93 @@ import mods.gregtech.recipe.Utils;
 
 //MATERIALS 8000-10000 RESERVED FOR NUCLEAR-RELATED ITEMS
 //MATERIALS 10000-20000 RESERVED FOR FLUIDS
-//MATERIALS 20000-30000 RESERVED FOR SOLIDS
-//MATERIALS 29000-30000 RESERVED FOR ELEMENTS
+//MATERIALS 20000-29000 RESERVED FOR SOLIDS
+//MATERIALS 29000-30000 RESERVED FOR ORES
+//MATERIALS 30000-31000 RESERVED FOR ELEMENTS
```

Comment:
It is unlikely that 1000 elements will be added. You could probably shrink this down to around 250 or less and still be very comfortable within the range.